src/tiroler-grostl.md

```diff
@@ -0,0 +1,33 @@
+# Tiroler Gröstl
+
+- ⏲️  Prep time: 15 min
+- 🍳 Cook time: 30 min
+- 🍽️ Servings: 5
+
+## Ingredients
+
+- 1 kg / 2 lbs potatoes
+- 2 red onions
+- 500 gr / 1 lbs porkchops
+- 400 gr / 14 ounce bacon (smoked)
+- 5 eggs
+- 3 cloves of garlic
+- Ample amount of chopped parsley
+- Marjoram and cumin to taste
+
+## Directions
+
+1. Clean potatoes and boil with skin.
+2. Chop onion, squeeze and mince the garlic.
+3. Dice pork and bacon.
+4. Slice cooked potatoes in thick wedges.
+5. Fry potato wedges in butter, add more butter while stirring.
+6. Add pork and bacon and bake until crispy.
+7. Add and stir the rest of the ingredients, except eggs.
+8. Fry the eggs seperately and serve on top.
+
+## Contribution
+
+ - Peter Wilbrink -- [printplaatreparatie.nl](https://printplaatreparatie.nl)
```

Comment:
```suggestion
 - Peter Wilbrink -- [website](https://printplaatreparatie.nl)
```

Reply:
Is this mandatory? It's not a personal website.